client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -98,13 +98,14 @@ export const middleware = handlers => store => next => {
 		// this guarantees that we don't double-dispatch
 		const nextActions = new Set();
 		const safeNext = a => nextActions.add( a );
-
-		handlerChain.forEach( handler => handler( store, action, safeNext ) );
+		let shouldIgnore = false;
+		const ignoreAction = () => shouldIgnore = true;
+		handlerChain.forEach( handler => handler( store, action, safeNext, ignoreAction ) );
```

Comment:
adding another argument to the handler API is something I want to actively avoid. it may be plausible to use the return value as some kind of indicator, but even with that idea I'd like to ponder it for a bit before making a decision. we're in the process of _removing_ `next`/`safeNext` so that the handlers only get `store` and `action`